templates/repo/issue/labels/edit_delete_label.tmpl

```diff
@@ -33,6 +33,16 @@
 				<div class="desc gt-ml-2 gt-mt-3 gt-hidden label-exclusive-warning">
 					{{svg "octicon-alert"}} {{.locale.Tr "repo.issues.label_exclusive_warning" | Safe}}
 				</div>
+				<br>
+			</div>
+			<div class="field label-is-archived-input-field">
+				<div class="ui checkbox">
+					<input class="label-is-archived-input" name="is_archived" type="checkbox">
+						<label>{{.locale.Tr "repo.issues.label_archive"}}</label>
+				</div>
+				<i class="gt-ml-2 gt-mt-2" data-tooltip-content={{.locale.Tr "repo.issues.label_archive_tooltip"}}>
+					{{svg "octicon-info" 13}}
```

Comment:
Why `13` ? The `svg` above uses default size.

Reply:
I think you are right  
<img width="689" alt="image" src="https://github.com/go-gitea/gitea/assets/80308335/11b29afc-49ca-4dc2-b502-ebd38d552891">